packages/openactive-broker-microservice/src/twoPhaseListeners/api.js

```diff
@@ -17,14 +17,8 @@ const { Listeners } = require('./listeners');
 function createOpportunityListenerApi(req, res) {
   if (!error400IfExpressParamsAreMissing(req, res, ['id'])) { return; }
   const { id } = req.params;
-  if (!error409IfListenerAlreadyExists(res, state.listeners.byOpportunityId, 'opportunities', id)) { return; }
-  if (state.listeners.byOpportunityId.has(id)) {
-    res.status(409).send({
-      error: `The @id "${id}" already has a listener registered. The same @id must not be used across multiple tests, or listened for multiple times concurrently within the same test.`,
-    });
-    return;
-  }
```

Comment:
This was duplicated logic. the `error409IfListenerAlreadyExists(..)` call already does the `if (state.listeners.byOpportunityId.has(id)) { .. }` check